docs/usage/administration/admin_api/federation.md

```diff
@@ -0,0 +1,81 @@
+# Federation API
+
+This API allows a server administrator to manage the federation.
+
+**Note**: This API is new, experimental and "subject to change".
+
+## List of destinations
+
+This API gets the current destination retry timing info for a remote server.
+
+The API is:
+
+A standard request with no filtering:
+
+```
+GET /_synapse/admin/v1/federation/destinations
+```
+
+A response body like the following is returned:
```

Comment:
Might be worth clarifying that a response body is not required, and is ignored?

Reply:
I am do not understand you. Does you mean an request body? IMO the `GET` has no request body.
Or do you mean, that `destinations` is empty if the server has no federation?

Reply:
> Does you mean an request body? IMO the GET has no request body.

Yes, that's what I meant. I hadn't considered that this was a GET.

For completeness, quoting [here](https://httpwg.org/specs/rfc7231.html#rfc.section.4.3.1):

> A payload within a GET request message has no defined semantics; sending a payload body on a GET request might cause some existing implementations to reject the request.